hub/apps/get-started/index.md

```diff
@@ -93,6 +93,30 @@ For more information about Windows Forms apps, see the following articles:
 * [API reference (.NET)](/dotnet/api/index)
 * [Enhancing Windows Forms apps](/dotnet/framework/winforms/advanced/)
 
+#### [React Native for Windows](#tab/rnw)
+[React Native](https://reactnative.dev) is a development platform from Facebook which allows building cross-platform apps.
+React Native for Windows brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you touse JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
+
+With React Native for Windows, you write most or all of your app code in JavaScript - or TypeScript - and the framework produces a native UWP XAML application. If your app needs to call a platform API, you can usually do so through one of the many [community modules](https://reactnative.directory), or a module does not yet exist, you can easily write a native module to expose it.
```

Comment:
Should this link to RNW's module documentation? https://microsoft.github.io/react-native-windows/docs/native-modules

Reply:
"....or [if] a module does not yet exist,.."

Reply:
it's linked to in the list at the bottom but I can link it here too/instead.

```suggestion
With React Native for Windows, you write most or all of your app code in JavaScript - or TypeScript - and the framework produces a native UWP XAML application. If your app needs to call a platform API, you can usually do so through one of the many [community modules](https://reactnative.directory), or a module does not yet exist, you can easily [write a native module to expose it](https://aka.ms/RNW-NativeModules).
```

Reply:
"...or [if] a module does not yet exist, you can..."

Reply:
Suggest that we link React Native Windows to [https://microsoft.github.io/react-native-windows/] since we link React Native out to Facebook's site.